tests/unit/config/test_ini.py

```diff
@@ -10,7 +10,7 @@
 
 
 @pytest.mark.skipif(not fs_supports_symlink(), reason="symlink is not supported")
-@pytest.mark.xfail(IS_PYPY and IS_WIN and sys.version_info[0:2] == (3, 9), reason="symlink is not supported")
+@pytest.mark.xfail(IS_PYPY and IS_WIN and sys.version_info[0:2] >= (3, 9), reason="symlink is not supported")
```

Comment:
Rather than skip the test can we understand why they don't support symlink.

Reply:
As [discussed in the PyPy](https://doc.pypy.org/en/latest/install.html?highlight=symlink) modern versions want users who want a system-wide PyPy to create a symlink in their bin dir so that PyPy can still find its library.

Reply:
Let's add link to that discussion.